build.gradle

```diff
@@ -76,7 +76,7 @@ githubRelease {
     owner = 'gradle'
     repo = 'common-custom-user-data-gradle-plugin'
     targetCommitish = 'main'
-    releaseName = gitHubReleaseName()
+    releaseName = version
```

Comment:
Set it as a provider rather than the resolved provider value, as you do for the body/release notes.

Reply:
Updated this to assign `releaseName = pluginVersion` instead of `project.version`. 

Though, `project.version` was previously being assigned to the provider returned by `version()`, so why would assigning `releaseName = project.version` use the resolved provider value instead of the provider itself? Just want to make sure I'm following your reasoning.